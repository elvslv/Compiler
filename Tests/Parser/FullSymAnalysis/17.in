var i, j: integer;
type my_int = integer;
function func(i : integer): my_int; forward;
begin
	if func(10) then
		i := j;
end.