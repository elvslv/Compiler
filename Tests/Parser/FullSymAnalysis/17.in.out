var I : INTEGER;
var J : INTEGER;
type MY_INT = INTEGER;

begin            
|                
|___if           
|   |            
|   |___()       
|   |   |        
|   |   |___FUNC 
|   |   |        
|   |   |___10   
|   |            
|   then         
|   |            
|   |___begin    
|       |        
|       |___:=   
|       |   |    
|       |   |___I
|       |   |    
|       |   |___J
|       |        
|       end      
|                
end              
